.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,32 @@
+---
+name: "✨ Feature request"
+about: Suggest a new feature or idea
+title: ''
+labels: "feature request, awaiting triage"
+assignees: ''
+
+---
+
+<!--
+
+  Before raising this feature request, please make sure you have checked that it isn’t already captured in:
+  - the community backlog: https://design-system.service.gov.uk/community/backlog/
```

Comment:
Sometimes we do ask people to promote issues from the backlog to the frontend repo, and if a feature has been 'lost' in the backlog raising it in the frontend repo seems like a good way to try and drive it forward. Is this definitely something we want to discourage?

Reply:
That's a good point, I think that's an oversight - we don't actually we want to discourage that. 

I wonder if we can say something along the lines of making sure to check the issue hasn't been raised already, but if it has to add your comment to that issue to help us understand when multiple users are experiencing the same problems/need the same features.

Do you have any ideas on this @EoinShaughnessy ?

Reply:
Sorry for the late response! This is what me and Vanita discussed adding:

"Your issue might already exist. If so, add a comment to the existing issue instead of creating a new one. You can find existing issues here:
  - all GOV.UK Frontend issues: https://github.com/alphagov/govuk-frontend/issues
- known validation error/warning list: https://github.com/alphagov/govuk-frontend/issues/1280#issuecomment-509588851"

Let me know if I should edit the window above to include this (or similar)!

Reply:
@36degrees I've updated the templates with Eoin's suggestion - do you think that makes it clearer?

Reply:
I think this file (`feature-request.md`) still needs updating?

Reply:
Done 👍